jdaviz/utils.py

```diff
@@ -259,3 +261,32 @@ def __call__(self):
 
     def reset(self):
         self.counter = -1
+
+
+def get_subset_type(subset):
+    """
+    Determine the subset type of a subset or layer
+
+    Parameters
+    ----------
+    subset : should have ``subset_state`` as an attribute, otherwise will return ``None``.
```

Comment:
```suggestion
    subset : glue.core.subset.Subset or glue.core.subset_group.GroupedSubset
        should have ``subset_state`` as an attribute, otherwise will return ``None``.
```